plugins/crane.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crane
+spec:
+  homepage: https://github.com/gocrane/kubectl-crane
+  shortDescription: "Easily interact with crane"
+  description: |
+    Kubectl plugin for crane, including recommendation and cost estimate,
```

Comment:
Is it possible to be a bit more descriptive here? What does this tool add on top of what's already at https://gocrane.io/docs/ ?

Even by looking at the website I can't tell what this plugin's functionality is. Ideally you should be able to say "This plugin lets you administer Crane by allowing you to do X, Y, Z". I recommend checking out descriptions of other plugins to see how they handle explain the functionality.

Reply:
Thanks, I will improve it soon.